src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -431,6 +431,14 @@ internal override bool IsDeclaredReadOnly
                     return true;
                 }
 
+                // can this accessor be made implicitly readonly?
```

Comment:
Please add a comment indicating why we are doing this. Something along the following lines would suffice:

Making a member implicitly `readonly` allows for valid C# 7.0 code to break PEVerify. Consider the following as a concrete example: 
struct S { 
  internal int Value { get; set; } 
  internal static readonly S StaticField = new S();
  static void M() { 
    Console.WriteLine(StaticField.Value);
  }
}

Marking Value as readonly implicitly means we would elide the copy in the Value call and hence fail PEVerify. To meet our goal of
having PEverify compatible code in 7.0 and before we will tie this behavior the language version it was introduced at

Reply:
Done